features/events/traps/src/test/java/org/opennms/netmgt/trapd/TrapdRoutingTest.java

```diff
@@ -0,0 +1,90 @@
+package org.opennms.netmgt.trapd;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.camel.builder.AdviceWithRouteBuilder;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.impl.JndiRegistry;
+import org.apache.camel.model.RouteDefinition;
+import org.apache.camel.test.junit4.CamelTestSupport;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.opennms.core.camel.JaxbUtilsUnmarshalProcessor;
+import org.opennms.core.test.OpenNMSJUnit4ClassRunner;
+import org.opennms.netmgt.config.trapd.TrapdConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+
+@RunWith(OpenNMSJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "classpath:/META-INF/opennms/emptyContext.xml" })
+public class TrapdRoutingTest extends CamelTestSupport {
```

Comment:
This test is incomplete. It needs to simulate the REST call and then test whether the REST routes can fetch the config file properly via REST.